manifests/piped/values.yaml

```diff
@@ -16,6 +16,9 @@ service:
   type: ClusterIP
   port: 9085
 
+# Optional additional labels to add to the Service
+# serviceLabels: {}
+
```

Comment:
I know in some cases, we don't want to add labels to all manifests, but just for service manifest.
How about moving this field to be an inner field of `service`?

``` yaml
service:
    enabled: true
    type: ClusterIP
    port: 9085
    additionalLabels: {}
```

Reply:
Certainly it's better!
I will change it.

Reply:
Thank you.